.gitpod.yml

```diff
@@ -0,0 +1,12 @@
+image:
```

Comment:
do you want to merge this change or its only part of your local configuration?

Reply:
This was a first test trying to overcome my local test-running-issue. Will remove that file later again, as it did not work like this out of the box.

Nevertheless it would be helpful to have a correct dockerfile for gitpod, as this would make developing for storj more easily. Different Topic, though. But as far as I understand the only Thing that's missing from the Jenkins-dockerfile are some Settings for the gitpod-user (like installing golang and Setting some env-flags).

Reply:
I've deleted the file